numpy/core/einsumfunc.py

```diff
@@ -1075,13 +1075,12 @@ def einsum(*operands, out=None, optimize=False, **kwargs):
     See Also
     --------
     einsum_path, dot, inner, outer, tensordot, linalg.multi_dot
-    einops :
-        similar verbose interface is provided by
+    einsum:
+        Similar verbose interface is provided by
         `einops <https://github.com/arogozhnikov/einops>`_ package to cover
         additional operations: transpose, reshape/flatten, repeat/tile,
         squeeze/unsqueeze and reductions.
-    opt_einsum :
-        `opt_einsum <https://optimized-einsum.readthedocs.io/en/stable/>`_
+        The `opt_einsum <https://optimized-einsum.readthedocs.io/en/stable/>`_
```

Comment:
Will a blank line break formatting? I think having one would provide a little more clarity.
```suggestion

        The `opt_einsum <https://optimized-einsum.readthedocs.io/en/stable/>`_
```

Reply:
No, it doesn't work. I have tried one and four blank lines, all no effect.